templates/security/security-standards.html

```diff
@@ -54,7 +54,7 @@ <h1>Security standards</h1>
       <div class="p-notification--information is-light">
         <div class="p-notification__content">
           <div class="p-notification__message">
-            FIPS 140-3 is now available for Ubuntu 22.04 LTS. <a href="/blog/fips-140-3-for-ubuntu-22-04lts" aria-label="Learn more about FIPS 140-3 for Ubuntu 22.04 LTS">Learn more.</a>
+            FIPS 140-3 is now available for Ubuntu 22.04 LTS. <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview" aria-label="Learn more about FIPS 140-3 for Ubuntu 22.04 LTS">Learn more.</a>
```

Comment:
```suggestion
            FIPS 140-3 is now available for Ubuntu 22.04 LTS. Learn more <a href="/blog/ubuntu-22-04-fips-140-3-modules-available-for-preview" aria-label="Learn more about FIPS 140-3 for Ubuntu 22.04 LTS">here</a>.
```
It appears like this in the [copydoc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit?tab=t.0)